OpenSourcePolitics/decidim-tou
# [Fix] - Scopes order in Meetings directory

#### :tophat: Description

This code is supposed to fix the order in meetings directory because the first level of indentation was well sorted numerically but when the scope had children, children were not sorted

#### :pushpin: Related Issues

- Related to [Notion](https://www.notion.so/opensourcepolitics/Toulouse-scopes-order-in-meetings-directory-0ad577a3f95c472caf8876c9502c4f4b?pvs=4)

#### Testing

* Log in as admin
* Access Backoffice
* Go to settings and modify some scopes to give them the format "number city" (ex: 1.0 Toulouse)
* Go back to the main page in front office
* Scroll down to meetings or change your URL to "/meetings"
* Check if the scopes you changed are well sorted alphanumerically
